components/select/select.tsx

```diff
@@ -0,0 +1,52 @@
+import { StylesConfig } from "react-select";
+import { StateManagerAdditionalProps } from "react-select/dist/declarations/src/useStateManager";
```

Comment:
Hnida: Potřebujeme tenhle import? Přijde mně, že ho nakonec nikde nepoužíváme.

Reply:
Máš pravdu, je to zbytečné, vyhodil jsem.